src/i18n/en/translation.en.json

```diff
@@ -535,18 +535,18 @@
     "visual-edit": {
       "avatar": {
         "title": "Avatar",
-        "description1": "{{width}} x {{height}} and supported formats recommended",
+        "description1": "Resolution: {{width}} width x {{height}} height (pixels)",
         "description2": "??"
       },
       "banner": {
         "title": "Banner",
-        "description1": "{{width}} x {{height}} and supported formats recommended",
-        "description2": "Used on the dashboard page"
+        "description1": "Resolution: {{width}} width x {{height}} height (pixels)",
+        "description2": "Usage: Banner on dashboard tab"
       },
       "banner-narrow": {
-        "title": "Banner narrow",
-        "description1": "{{width}} x {{height}} and supported formats recommended",
-        "description2": "Used on the cards"
+        "title": "Banner for cards",
```

Comment:
The wording is duplicated "Banner for cards" and "Banner on cards". I think we could stick to only title and `descriptio1`.